src/app/sign-in/components/errors/print-errors/print-errors.component.scss

```diff
@@ -5,3 +5,7 @@
 .orcid-error {
   margin-top: 20px !important;
 }
+
+.resend {
```

Comment:
@DanielPalafox This is not a bad solution, but it won't work on Arabian language, can we use the "preserve white spaces" on the component as mentioned here https://trello.com/c/WBV1QJAM/6702-qa-signin-issues-with-claiming-record-process#comment-5ea1ad533c812657d48b078a